src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -1151,6 +1171,141 @@ class boss_icehowl : public CreatureScript
         }
 };
 
+class spell_gormok_jump_to_hand : public SpellScriptLoader
+{
+public:
+    spell_gormok_jump_to_hand() : SpellScriptLoader("spell_gormok_jump_to_hand") { }
+
+    class spell_gormok_jump_to_hand_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_gormok_jump_to_hand_AuraScript);
+
+        bool Validate(SpellInfo const* /*spell*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_RIDE_PLAYER))
+                return false;
+            return true;
+        }
+
+        bool Load() override
+        {
+            if (GetCaster() && GetCaster()->GetEntry() == NPC_SNOBOLD_VASSAL)
+                return true;
+            return false;
+        }
+
+        void OnRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            if (Unit* caster = GetCaster())
+            {
+                if (CreatureAI* gormokAI = GetTarget()->ToCreature()->AI())
+                {
+                    if (Unit* target = gormokAI->SelectTarget(SELECT_TARGET_RANDOM, 0, SnobolledTargetSelector(GetTarget())))
+                    {
+                        gormokAI->Talk(EMOTE_SNOBOLLED);
+                        caster->GetAI()->DoAction(ACTION_ACTIVE_SNOBOLD);
+                        caster->CastSpell(target, SPELL_RIDE_PLAYER, true);
+                    }
+                }
+            }
+        }
+
+        void Register() override
+        {
+            AfterEffectRemove += AuraEffectRemoveFn(spell_gormok_jump_to_hand_AuraScript::OnRemove, EFFECT_0, SPELL_AURA_CONTROL_VEHICLE, AURA_EFFECT_HANDLE_REAL);
+        }
+    };
+
+    AuraScript* GetAuraScript() const override
+    {
+        return new spell_gormok_jump_to_hand_AuraScript();
+    }
+};
+
+class spell_gormok_ride_player : public SpellScriptLoader
```

Comment:
You might want to test this one a bit in combination with players DCing while they have the aura. It's a bit of an uncommon aura, so just to be safe.